src/settings/MappingProfiles/detailsSections/view/InvoiceDetailsSection/InvoiceLineInformation.js

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  Col,
+  Row,
+  KeyValue,
+  NoValue,
+} from '@folio/stripes/components';
+
+import { ProhibitionIcon } from '../../../../../components';
+import { ViewRepeatableField } from '../ViewRepeatableField';
+
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  getFieldValue,
+  transformSubfieldsData,
+  renderCheckbox,
+} from '../../utils';
+import { mappingProfileFieldShape } from '../../../../../utils';
+
+export const InvoiceLineInformation = ({ invoiceLineMappingDetails }) => {
+  const noValueElement = <NoValue />;
+  const prohibitionIconElement = fieldName => <ProhibitionIcon fieldName={fieldName} />;
+
+  const descriptionValue = getFieldValue(invoiceLineMappingDetails, 'description', 'value');
+  const POLineNumberValue = getFieldValue(invoiceLineMappingDetails, 'poLineId', 'value');
```

Comment:
Name the variable with a lowercase, please

Reply:
done